kalite/distributed/static/js/distributed/content/views.js

```diff
@@ -6,6 +6,7 @@ var VideoModels = require("video/models");
 var $script = require("scriptjs");
 
 var ContentBaseView = require("./baseview");
+var ExerciseModels = require("exercises/models");
```

Comment:
One consequence of this approach, is that Exercise content data is now accessible through the same API endpoint as other content. This means we can unify Exercises with all the other content type in the front end.